src/azure/policyHelper.ts

```diff
@@ -89,6 +89,11 @@ export async function getAllPolicyRequests(): Promise<PolicyRequest[]> {
         }
       }
     }
+
+    // There were errors while getting policies. We will not proceed further.
+    if (allPolicyDetails.length != policyRequests.length) {
```

Comment:
Shouldn't the behaviour be the following -
Try applying other policies and report this error in the logs/summary. The overall action status should be failed.

Should we discuss this with the PMs once?

Reply:
Discussed with PM.
We might end up in a bad state if we try to skip applying some policies.